execution/function/operator.go

```diff
@@ -29,36 +30,56 @@ type functionOperator struct {
 
 	call         functionCall
 	scalarPoints [][]float64
+	model.OperatorTelemetry
+}
+
+func SetTelemetry(t model.OperatorTelemetry, opts *query.Options) {
+	t = &model.NoopTelemetry{}
+	if opts.EnableAnalysis {
+		t = &model.TrackedTelemetry{}
+	}
```

Comment:
This causes lint to fail. Let's return simply,
```suggestion
func SetTelemetry(opts *query.Options) model.OperatorTelemetry {
	if opts.EnableAnalysis {
		return &model.TrackedTelemetry{}
	}
	return &model.NoopTelemetry{}
```